l10n/configs/pontoon.toml

```diff
@@ -148,3 +148,11 @@ locales = [
 [[paths]]
     reference = "en/**/*.ftl"
     l10n = "{locale}/**/*.ftl"
+
+[[paths]]
```

Comment:
I think this new definition should be above the one above it which appears to be a catch-all definition. I'm honestly not that familiar with these config files either, but to me it looks like order is important.

Reply:
Ah ha! Found the docs for this file format https://moz-l10n-config.readthedocs.io/en/latest/fileformat.html#paths

Reply:
that sounds like a good idea. Updating.

Reply:
Cool. Looks good to me then 👍